proxy/http/HttpConfig.cc

```diff
@@ -900,6 +900,59 @@ register_stat_callbacks()
                      (int)http_sm_finish_time_stat, RecRawStatSyncSum);
 }
 
+static bool
+set_negative_caching_list(const char *name, RecDataT dtype, RecData data, HttpConfigParams *c, bool update)
```

Comment:
Is this called from anywhere else? Maybe it should be folded in to `negative_caching_list_cb`.

Reply:
`negative_caching_list_cb ` is only called when reloading. `set_negative_caching_list` is called once in the first place.